### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by VenusGirl**
Korean Products Name

### CODE REVIEW

File: src/OfficeToolPlus/Dictionaries/ProductsName/ko-kr.xaml [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    <sys:String x:Key="SkypeforBusinessEntryRetail">Business Basic 2016용 Skype</sys:String>
+    <sys:String x:Key="SkypeforBusinessRetail">Business 2016용 Skype</sys:String>
+    <sys:String x:Key="SkypeforBusinessVolume">Business 2016용 Skype - 볼륨 라이선스</sys:String>
+    <sys:String x:Key="SkypeServiceBypassRetail">Business 2016용 Skype</sys:String>
+    <sys:String x:Key="Standard2019Retail">Office Standard 2019</sys:String>
+    <sys:String x:Key="Standard2019Volume">Office Standard 2019 - 볼륨 라이선스</sys:String>
+    <sys:String x:Key="Standard2021Retail">Office Standard 2021</sys:String>
+    <sys:String x:Key="Standard2021Volume">Office LTSC Standard 2021 - 볼륨 라이선스</sys:String>
+    <!-- If you are translating to LTR language, please remove '&#x200e;', otherwise keep it. -->
+    <sys:String x:Key="StandardSPLA2021Volume">Office LTSC Standard 2021 (SPLA)&#x200e; - 볼륨 라이선스</sys:String>
```
Comment (YerongAI) [inline src/OfficeToolPlus/Dictionaries/ProductsName/ko-kr.xaml:102]: `&#x200e;` can be removed.

File: src/OfficeToolPlus/Dictionaries/ProductsName/ko-kr.xaml [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    <sys:String x:Key="ProjectStd2021Volume">Project Standard 2021 - 볼륨 라이선스</sys:String>
+    <sys:String x:Key="ProjectStdRetail">Project Standard 2016</sys:String>
+    <sys:String x:Key="ProjectStdVolume">Project Standard 2016 - 볼륨 라이선스</sys:String>
+    <sys:String x:Key="ProjectStdXVolume">Project Standard 2016 - 볼륨 라이선스</sys:String>
+    <sys:String x:Key="ProPlus2019Retail">Office Professional Plus 2019</sys:String>
+    <sys:String x:Key="ProPlus2019Volume">Office Professional Plus 2019 - 볼륨 라이선스</sys:String>
+    <sys:String x:Key="ProPlus2021Retail">Office Professional Plus 2021</sys:String>
+    <sys:String x:Key="ProPlus2021Volume">Office LTSC Professional Plus 2021 - 볼륨 라이선스</sys:String>
+    <!-- If you are translating to LTR language, please remove '&#x200e;', otherwise keep it. -->
+    <sys:String x:Key="ProPlusSPLA2021Volume">Office LTSC Professional Plus 2021 (SPLA)&#x200e; - 볼륨 라이선스</sys:String>
```
Comment (YerongAI) [inline src/OfficeToolPlus/Dictionaries/ProductsName/ko-kr.xaml:79]: `&#x200e;` can be removed.

File: src/OfficeToolPlus/Dictionaries/ProductsName/ko-kr.xaml [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    <sys:String x:Key="HomeStudent2019Retail">Office Home 및 Student 2019</sys:String>
+    <sys:String x:Key="HomeStudent2021Retail">Office Home 및 Student 2021</sys:String>
+    <sys:String x:Key="HomeStudentRetail">Office Home 및 Student 2016</sys:String>
+    <sys:String x:Key="HomeStudentVNextRetail">Office Home 및 Student 2016</sys:String>
+    <sys:String x:Key="MondoRetail">Office Mondo 2016</sys:String>
+    <sys:String x:Key="MondoVolume">Office Mondo 2016 - 볼륨 라이선스</sys:String>
+    <sys:String x:Key="O365BusinessRetail">비즈니스용 Microsoft 365 앱</sys:String>
+    <sys:String x:Key="O365EduCloudRetail">Microsoft 365</sys:String>
+    <!-- If you are translating to LTR language, please remove '&#x200e;', otherwise keep it. -->
+    <sys:String x:Key="O365HomePremRetail">Microsoft 365 (가족 및 개인)&#x200e;</sys:String>
```
Comment (YerongAI) [inline src/OfficeToolPlus/Dictionaries/ProductsName/ko-kr.xaml:33]: `&#x200e;` can be removed.

**Review by YerongAI — CHANGES_REQUESTED** [review]

**Review by VenusGirl — COMMENTED** [review]
Fixed

**Review by VenusGirl — COMMENTED** [review]
Fixed

**Review by YerongAI — APPROVED** [review]

**Pull request merged**